packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -83,20 +107,20 @@ export class Checkbox extends BaseComponent<ICheckboxProps, ICheckboxState> impl
           ref={ this._resolveRef('_checkBox') }
           id={ this._id }
           name={ name || this._id }
-          className={ css('ms-Checkbox-input', styles.input) }
```

Comment:
Make sure that taking away the hardcoded classnames doesn't break any unit tests